api/v1/helper.go

```diff
@@ -300,6 +306,28 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 							return true
 						}
 
+						if strings.EqualFold(ifaceStatus.LinkType, consts.LinkTypeETH) {
```

Comment:
nit: any chance to have one check to avoid duplication ?

e.g if (ETH && RDMA) || IB { ....}

Reply:
Yeap, makes sense. Addressing.